src/components/TodoListItem.js

```diff
@@ -15,4 +16,9 @@ const TodoListItem = ({ item, onRemoveClicked }) =>
         </div>
     </li>;
 
+TodoListItem.propTypes = {
+    item: PropTypes.object.isRequired,
```

Comment:
Stretch Goal:
What is `item` an object of (what's its shape)?

Reply:
item: PropTypes.shape({
        id: PropTypes.string.isRequired,
        title: PropTypes.string.isRequired,
    })

Reply:
Thanks for the tip! I went ahead and added the specific types.